doc/interfaces/gibbs_unringing_flow.rst

```diff
@@ -69,10 +69,10 @@ the Gibbs unringing effect.
 
 References
 ----------
-.. [1] Veraart, J., Fieremans, E., Jelescu, I.O., Knoll, F., Novikov, D.S.,
-       2015. Gibbs Ringing in Diffusion MRI. Magn Reson Med 76(1): 301-314.
-       https://doi.org/10.1002/mrm.25866
-.. [2] Perrone, D., Aelterman, J., Pižurica, A., Jeurissen, B., Philips, W.,
-       Leemans A., 2015. The effect of Gibbs ringing artifacts on measures
-       derived from diffusion MRI. Neuroimage 120, 441-455.
-       https://doi.org/10.1016/j.neuroimage.2015.06.068.
\ No newline at end of file
+.. [Veraart15] Veraart, J., Fieremans, E., Jelescu, I.O., Knoll, F., Novikov, D.S.,
+               2015. Gibbs Ringing in Diffusion MRI. Magn Reson Med 76(1): 301-314.
+               https://doi.org/10.1002/mrm.25866
```

Comment:
Why are the whitespaces introduced here but not in the other references? I think we should choose and stick one style if both are rendered correctly.

Reply:
Agreed! Addressed in 54907a3